Resources/Prototypes/_LostParadise/Reagents/Consumable/Drink/drinks.yml

```
@@ -0,0 +1,112 @@
+  color: "#237e66"
+  recognizable: true
+  metabolisms:
+    Drink:
+      effects:
+      - !type:SatiateThirst
+        factor: 2
+      - !type:AdjustReagent
+        reagent: Theobromine
+        amount: 0.05
+
+- type: reagent 
```

yamllint

[error] trailing spaces

(trailing-spaces)